docs/REST API/rest-api.md

```diff
@@ -2976,9 +2976,9 @@ Request (`filters` is specified and empty, to delete all hosts):
 
 `Status: 200`
 
-### Get host's Google Chrome profiles
+### Get host's user profiles
```

Comment:
I like "human-host" terminology. I think "user" (...user-host mapping) is a term that's mostly used in this scenario but could also be confusing since we refer to Fleet users as "users".